maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -175,10 +181,20 @@ class Orchestra(
             initJsEngine(config)
         }
 
-        initAndroidChromeDevTools(config)
+        if (!coroutineContext.isActive) {
+            logger.info("Flow cancelled, skipping initAndroidChromeDevTools...")
+        } else {
+            initAndroidChromeDevTools(config)
+        }
 
         commands
             .forEachIndexed { index, command ->
+                if (!coroutineContext.isActive) {
```

Comment:
I'm thinking how this should look like for cli, should there be any controls there? I believe not useful, because users usually press control + c to break from flows.

Although I'm not sure about CI

Reply:
ctrl+c kills the entire Java process, so it's not gonna impact any of this. Coroutine cancellation is for when you have a process still running and you want to cancel it